src/main/java/config/WebConfig.java

```diff
@@ -0,0 +1,23 @@
+package config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan("com.cinema.controller")
+public class WebConfig {
+
+    @Bean
+    public InternalResourceViewResolver getResolver() {
```

Comment:
if you don't have .jsp in your project, than we can remove this view resolver) and dependencies which are related to jsp from pom.xml

Reply:
Done